_tutorials/preferences-api.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: develop
+title: Using the Preferences API
+
+summary: The Fyne toolkit includes a Preferences API that makes it easy for developers to store values locally on the computer and then grab them again when necessary.
+
+order: 50
+redirect_from: /develop/preferences-api.html
+---
+
+## Using the Preferences API
+
+---
+
+Writing filesystem code is a common task for application developers and, to make that easier, Fyne has an API for storing values on the filesystem in a clean and understandable way while Fyne handles the complex parts for you.
+
+Lets start with the setup of the API. It is part of the [Preferences](https://pkg.go.dev/fyne.io/fyne?tab=doc#Preferences) interface where storage and loading functions exist for values of Bool, Float, Int and String. They each consist of three different functions, one for loading, one loading with a fallback value and lastly, one for storing values. An example of the three functions and their behaviour can be seen below for the String type:
+```go
+// String looks up a string value for the key
+String(key string) string
+// StringWithFallback looks up a string value and returns the given fallback if not found
+StringWithFallback(key, fallback string) string
+// SetString saves a string value for the given key
+SetString(key string, value string)
+```
+
+These functions can be accessed through the created application variable and it is necessary that it is created with a unique ID. This means that the application will need to be created using `app.NewWithID()` to have its own place to store values. the can roughly be used like the example below:
+```go
+a := app.NewWithID("fyne-preferences-api-tutorial")
+[...]
+a.Preferences().SetBool("Boolean?", true)
```

Comment:
I don't think we should encourage special characters in these fields. we sometimes need to delegate to underlying preference storage and I'm not certain we check for all these eventualities yet...

Reply:
Sure thing. Just thought it was kind of funny when I wrote it. Will remove it :)